lib/case.js

```diff
@@ -73,39 +81,54 @@ module.exports = (metatests) => {
           sResult = JSON.stringify(result);
         }
 
+        let msg;
         if (targetType === 'method') {
-          targetValue = target;
-          if (target === null) targetValue = '';
-          else targetValue = JSON.stringify(target);
-          msg = (
-            targetType + ' of ' + className + ': ' +
-            targetValue + '.' + methodName
-          );
+          const targetValue = target === null ? '' : JSON.stringify(target);
+          msg = `${targetType} of ${className}: ${targetValue}.${methodName}`;
         } else {
-          msg = targetType + ' ' + functionName;
+          msg = `${targetType} ${functionName}`;
```

Comment:
`targetType + ' ' + functionName` looks better in this certain case, but exact same length :smile:

Reply:
I'd prefer
```
msg = `${targetType} ${functionName}\`;
```